Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,20 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json? && !args.quiet? && $stdout.tty?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
+        ohai "Generating JSON"
+        progress = ProgressBar.create(
+          total:         total_formulae,
+          progress_mark: "#",
+          format:        " %t: |%B| %c/%C ",
```

Comment:
```suggestion
          total:          total_formulae,
          progress_mark:  "#",
          remainder_mark: ".",
          format:         " %t: [%B] %c/%C ",
```

I was updating a server yesterday and found some inspiration. This version takes some cues from Ubuntu's progress bar, with square brackets at the start/end and periods as remainder marks:

```
 Progress: [###..........................................................] 1/17
```

I think this is the last time I'll tweak this. What do you think about this format?

Reply:
Looks good!